.github/workflows/restore-app-main-db.yml

```diff
@@ -49,7 +52,9 @@ jobs:
 
     - uses: azure/login@v2
       with:
-        creds: ${{ secrets.azure-credentials || secrets.AZURE_CREDENTIALS }}
+        client-id: ${{ secrets.AZURE_CLIENT_ID }}
```

Comment:
I have added `migration` to the list of environments available to select for this workflow, will that have any affect on the work here?  Does anything else need to be set up for the `migration` environment?